tests/components/probe_plus/test_config_flow.py

```diff
@@ -0,0 +1,116 @@
+"""Test the config flow for the Probe Plus."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant.components.probe_plus.const import DOMAIN
+from homeassistant.config_entries import SOURCE_BLUETOOTH, SOURCE_USER
+from homeassistant.const import CONF_ADDRESS
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+from homeassistant.helpers.service_info.bluetooth import BluetoothServiceInfo
+
+from tests.common import MockConfigEntry
+
+service_info = BluetoothServiceInfo(
+    name="FM210 aa:bb:cc:dd:ee:ff",
+    address="aa:bb:cc:dd:ee:ff",
+    rssi=-63,
+    manufacturer_data={},
+    service_data={},
+    service_uuids=[],
+    source="local",
+)
+
+
+@pytest.fixture
+def mock_discovered_service_info() -> Generator[AsyncMock]:
+    """Override getting Bluetooth service info."""
+    with patch(
+        "homeassistant.components.probe_plus.config_flow.async_discovered_service_info",
+        return_value=[service_info],
+    ) as mock_discovered_service_info:
+        yield mock_discovered_service_info
+
+
+async def test_form(
+    hass: HomeAssistant,
+    mock_setup_entry: AsyncMock,
+    mock_discovered_service_info: AsyncMock,
+) -> None:
+    """Test we get the form."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER}
+    )
+    assert result["type"] is FlowResultType.FORM
+    assert result["step_id"] == "user"
+
+    user_input = {
+        CONF_ADDRESS: "aa:bb:cc:dd:ee:ff",
+    }
+    result2 = await hass.config_entries.flow.async_configure(
```

Comment:
Keep it as `result`, so it chronology changes and you keep the current state.
```suggestion
    result = await hass.config_entries.flow.async_configure(
```

Reply:
Thanks, first time writing tests I must admit